contracts/gateway/GatewayBase.sol

```diff
@@ -148,6 +152,7 @@ contract GatewayBase is Organized {
      */
     mapping(address => bytes32) private outboxActiveProcess;
 
+    CircularBufferUintLib.BufferUint storageRootBuffer;
```

Comment:
the name `storageRootBuffer` is badly named because it is storing uint, not storage root